docs/CHILI-GraFx/concepts/federated-single-sign-on/saml/index.md

```diff
@@ -1,24 +1,79 @@
-# Federated Single sign-on: SAML
+# Federated Single Sign On: SAML
 
-This document lists the info for your SAML Identity Providers in the CGX Platform.
+This document lists the steps required for setting up Federated Single Sign On from your SAML Identity Provider ("IDP") to CHILI GraFx.
 
-## CHILI GraFx will need
+## 1. Configure your IDP
 
-- Sign in URL 
-- X509 Signing Certificate 
-- Claim names for this information: 
-	- Email (default is email)
-	- First Name (default is given_name)
-	- Last Name (default is family_name)
-	- Group memberships (default is https://chili-publish.com/CGXGroups)
-- Identity attribute 
+Create a SAML application in your IDP.
 
-## We will provide you
+To do this, you will need a few details:
 
-- Authentication request signing certificate: Provided upon request. 
-- SP ID: Provided upon request. 
-- The identity attribute must be the user’s corporate email address 
+- Redirect URI: `https://login.chiligrafx.com/login/callback`
+- Service Provider ID
+  - The name GraFx will identify itself as on your IDP
+  - This will be provided by our Client Success team
+
+Please configure your IDP to provide at least following attributes in SAML responses:
+
+| **Attribute name**                        | **Description**                                           |
+| ------------------------------------- | --------------------------------------------------------- |
+| `email`                               | The email address of the user                             |
+| `given_name`                          | The given name of the user                                |
+| `family_name`                         | The family name of the user                               |
+| `https://chili-publish.com/CGXGroups` | A list of UUIDs of the GraFx groups the user should be in |
+
+Also, please configure the application to use the **email address** as the `NameID` in SAML responses.
```

Comment:
nitpicky - it doesn't flow well 
not completely sure about it but, from setup perspective iirc, usually setting Name ID was before additional attribute mapping, so it would be better to have it before Attributes - to mimic setup flow
from reading perspective - it mixes up 3 ways of communicating required setup lists for info from CS (Service Provider ID etc.), table for Attributes, text for email address as Name ID
it is "fatiguing" to understand since you kinda need to switch contexts
it would be best to have 1 ways (ex. table), but multiple of them ex. Table for "base setup" with "info from CS" and NameID and table for attributes